tkhomich/04_css/CSS_Python_2/Python 2.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<HTML>
+<HEAD>
+    <META charset="utf-8">
+    <TITLE>Программа на Python 2</TITLE>
+</HEAD>
+<BODY>
+     <!-- Вариант 2 -->
+     <style media="screen">
```

Comment:
What tag style does here? It should be in tag head

Reply:
Updated. Moved to Head.